automation/Extension/firefox/feature.js/callstack-instrument.js

```diff
@@ -0,0 +1,19 @@
+/*
+    This script catches the stacktraces of JavaScript that ran on a crawled site and made a WebRequest
```

Comment:
nit: 2-space indent throughout this file (do we have linting for this)

Reply:
I don't think we have any linting set up for .js files, I've only seen linters for the .ts and .py files.